README.md

```diff
@@ -100,19 +100,20 @@ For more information about parameters that can be configured with the AWS JDBC D
 
```

Comment:
Can you please fix a typo in the README
```
| RDS Custom Cluster | `jdbc:mysql:aws://db-identifier.cluster-custom-XYZ.us-east-2l.rds.amazonaws.com:3306`      |    None | *Initial connection:* any DB instance in the custom DB cluster<br/>*Failover 
```
Should be `us-east-2` instead of `us-east-2l`